composer.json

```
@@ -9,13 +9,14 @@
     "license": "MIT",
     "require": {
         "php": "^8.0",
-        "sylius/sylius": "^1.12",
+        "sylius/sylius": "~1.12.15",
         "sylius/mailer-bundle":  "^1.8 || ^2.0@beta",
```

As 2.0 is released or even we could remove that dependency as it should exist in sylius/sylius
```suggestion
        "sylius/mailer-bundle":  "^1.8 || ^2.0",
```